lstchain/reco/reconstructor.py

```diff
@@ -628,114 +616,86 @@ def array_times_template(array, ti, template, is_high_gain):
         # more than 10^-6. The limits are approximated by 2 broken linear
         # function obtained for 0 crosstalk.
         # The choice of kmin and kmax is currently not done on a pixel basis
-
-        # Alternative kmin, kmax and mask determination, faster?
-        # Results are fully compatible
-        mask = (mu <= self.n_peaks/1.096 - 47.8)
-        if len(mu[mask]) == 0:
+        mask_LL = (mu <= self.n_peaks/1.096 - 47.8)
```

Comment:
Can you please add comment explaining numerical constants used here and below?

Reply:
The explanation is right above. With the numerical constants being the parameters of the broken linear functions obtained "by hand". I started looking at the results of using a polynomial fitted using numpy.polyfit (degree 3 seems to work well). It should be the work of another PR, especially if the dependence to the crosstalk is to be included.